src/App.js

```diff
@@ -1,14 +1,45 @@
-import React, { Component } from "react";
-import { View, Text } from "react-native";
-
-class App extends Component {
-	render() {
-		return(
-			<View style={{justifyContent:"center", flex:1}}>
-				<Text style={{alignSelf: "center"}}>Ask-SCoRe App</Text>
-			</View>
-		);
-	}
+import React, { Component } from 'react';
+import 'react-native-paper';
+import OnboardingScreen from './screens/OnboardingScreen/OnboardingScreen';
+import StartScreen from './screens/StartScreen/StartScreen';
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import AsyncStorage from '@react-native-community/async-storage';
+import { useEffect } from 'react';
+
+const AppStack = createStackNavigator();
+
+const App = () => {
+    const [isFirstLaunch, setIsFirstLaunch] = React.useState(null);
+
+    useEffect(() => {
+        AsyncStorage.getItem('alreadyLaunched').then(value => {
+            if(value == null) {
+                AsyncStorage.setItem('alreadyLaunched', 'true');
```

Comment:
Let's make this variable unique. Something like the package name(something like com.askscore.alreadyLaunched). Use a config file to store the package name and import it in this file and replace variable with that.

Reply:
Done!